### PR TITLE
DOCS: Add reminder to create github label after adding new provider

### PR DESCRIPTION
**Pull request by tlimoncelli**

### CODE REVIEW

File: documentation/writing-providers.md [hunk 1]
```diff
@@ -331,4 +331,5 @@ submit a PR if you haven't already.
 ## Step 16: After the PR is merged
 
 1. Close any related GitHub issues.
+2. [Create an issue (feature request)](https://github.com/StackExchange/dnscontrol/issues/new/choose) titled "Add label for PROVIDERNAME" with the text "Please create  the GitHub label 'provider-PROVIDERNAME'".
```
Comment (cafferata) [inline documentation/writing-providers.md:334]: You might consider including the title in the GitHub link:

```suggestion
2. [Create an issue (feature request)](https://github.com/StackExchange/dnscontrol/issues/new?title=Add%20label%20for%20PROVIDERNAME) with the text "Please create  the GitHub label 'provider-PROVIDERNAME'".
```

Comment (cafferata) [inline documentation/writing-providers.md:334]: **Style**: There is a (unnecessary) double space:

```suggestion
2. [Create an issue (feature request)](https://github.com/StackExchange/dnscontrol/issues/new/choose) titled "Add label for PROVIDERNAME" with the text "Please create the GitHub label 'provider-PROVIDERNAME'".
```

**Review by cafferata — APPROVED** [review]
Great idea to include this! Another small possible improvement.

**Comment by tlimoncelli** [comment]
Thanks!

**Pull request merged**